### ISSUE TITLE
App crashes on attempt to buy ENS

### ISSUE THREAD

**Issue by igor-sirotin** [status-im/status-desktop#9264]
# Bug Report

This bug was found while testing #9036.
Check this comment for details: https://github.com/status-im/status-desktop/pull/9036#issuecomment-1401047536

Though I tested it on d980c177a4467cbb2fa9277dd6651c369cf783cc, which is behind my changes. 
status-go is also behind my changes, it points to status-im/status-go@ea46779e98708616201728e422253564809043dc.

## Steps to reproduce

1. Create a new account
2. Enable wallet
3. Switch to testnet
4. Go to "ENS usernames" settings
5. Input a desired username, note "Username available" label
6. Click the arrow button, the app crashes. 

## Details

The app crashes when trying to find token by symbol `STT`.
That happens because token-service `self.tokens` empty.
It's empty because during initialization `backend.getCachedPrices()` fails with error message 
`key not found: AKROUSD`.

It seems to be broken after https://github.com/status-im/status-desktop/pull/9093.

```shell
ERR 2023-01-24 16:50:34.666+03:00 Error finding token by symbol              topics="token-service" tid=6432963 file=service.nim:110 msg="key not found: 5"
Traceback (most recent call last)
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/src/nim_status_client.nim(180) nim_status_client
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/src/nim_status_client.nim(177) mainProc
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/vendor/nimqml/src/nimqml/private/qguiapplication.nim(29) exec
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/vendor/nimqml/src/nimqml/private/qobject.nim(49) qobjectCallback
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/vendor/nimqml/src/nimqml/private/qobject.nim(34) onSlotCalled
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/vendor/nimbus-build-system/vendor/Nim/lib/core/macros.nim(582) onSlotCalled
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/src/app/modules/main/profile_section/ens_usernames/view.nim(118) getSNTBalance
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/src/app/modules/main/profile_section/ens_usernames/io_interface.nim(75) getSNTBalance
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/src/app/modules/main/profile_section/ens_usernames/module.nim(322) getSNTBalance
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/src/app/modules/main/profile_section/ens_usernames/controller.nim(132) getSNTBalance
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/src/app_service/service/ens/service.nim(388) getSNTBalance
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/src/app_service/service/token/dto.nim(64) addressAsString
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/vendor/nim-web3/web3/conversions.nim(91) $
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/vendor/nim-task-runner/vendor/nim-stew/stew/byteutils.nim(135) toHex
/Users/igorsirotin/Repositories/Status/status-desktop-cpp/vendor/nim-task-runner/vendor/nim-stew/stew/byteutils.nim(122) toHexAux
```

**Comment by igor-sirotin** [comment]
@dlipicar, could you please check this one out?

**Comment by igor-sirotin** [comment]
@dlipicar, I've fixed everything for my needs in #9330 
But please, check out my changes, you will see that there're more things to fix, it's easily seen by code.
Please, contact me when you're around 🙂

**Comment by dlipicar** [comment]
@igor-sirotin added a small comment.
Thanks for taking care of this while I was away!
